test/ConfigFiles/ConfigFIle_test.e3d

```diff
@@ -0,0 +1,83 @@
+-------------------------------------------------------------------------------
+%%%%%%%%%%%%%%%%     EULER3D Software Input file    %%%%%%%%%%%%%%%%%%%%%
+Author : AMIN OULED-MOHAMED
+Simulation Title : TESTING
+Date : 04-02-2020
+Comments : For testing purposes (Not Stable)
+-------------------------------------------------------------------------------
+
+START
+
+------------------- PRE-PROCESSING CONTROL -------------------
+#Path to partitioned mesh files (from executable directory)
+PARTITION_FILES= 4
+../../test/mesh/StructuredBlock_8#0.par
```

Comment:
This syntax might be a bit overkill. We could change it to ../../test/mesh/StructuredBlock_8_p#.par
During writing/reading we can change the `#` for the partition ID

Reply:
Naming isn't really important for the functioning of the software. It's up to the user whether to name it, for example, StructuredBlock_8_p0.par or StructuredBlock_8#0.par, or even 0.par

Reply:
This is not exactly what i meant. Let's say the user wants to make 10 partitions named p0.part, p1.part, p2.part....

In the current format, it would be formatted like so:
```
p0.part
p1.part
p2.part
p3.part
p4.part
p5.part
p6.part
p7.part
p8.part
p9.part
```

My proposition is to format it like so:
```
p#.part
```

I think it would make the config file more readable when there are many partitions.

Reply:
Done in latest commit.